submissions/mubarakOlaoluwa/Database/init.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE food_bag;
+
+/* This is the table for the first success criteria (food type) */
+USE food_bag;
+CREATE TABLE food_table (
+	id INT NOT NULL AUTO_INCREMENT,
+    type VARCHAR(50) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+/*This inserts the various food types into the food_table database*/
+INSERT INTO food_table (type)
+	VALUES ('Fast food'),
+		   ('Vegetable & Fruits'),
+           ('Drinks & Cocktails'),
+           ('Resturants');
+
+/* This is the table for the second success criteria (vendors food) */
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (food_id) REFERENCES food_table(id)
+);
+
+/*This is the table for the third success criteria (admins table)*/
+CREATE TABLE admins(
+	id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(50) NOT NULL,
+    type INT NOT NULL,
+    email_address VARCHAR(50) NOT NULL,
+    phone_number INT NOT NULL,
```

Comment:
For a mobile number that starts with +2348078888, you will agree with me that, + wont be accepted as an integer or 08033003800, the 0 will be unaccepted too, likewise VARCHAR means capita letter O can be accepted as 0.

Just wanted you to understand the datatype you chose and what it entails